### PR TITLE
Add SuperFlexibleSoftware.SzArchiver 0.84

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116409&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
+PackageLocale: en-US
+Publisher: Super Flexible Software GmbH & Co. KG
+PackageName: SzArchiver
```
Comment (stephengillie) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml:8]:
```suggestion
PackageName: SzArchiver
PackageUrl: https://www.syncovery.com/szarchiver/
```
Hi @chausner,

This `PackageUrl` helps me to validate that the InstallerUrl is valid, and will let interested users know where to learn more about this package.

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
```
Comment (Trenly) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.installer.yaml:5]:
```suggestion
PackageVersion: '0.84'
```

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
+PackageLocale: en-US
+Publisher: Super Flexible Software GmbH & Co. KG
```
Comment (Trenly) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml:7]:
```suggestion
Publisher: Super Flexible Software
```

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
```
Comment (Trenly) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml:5]:
```suggestion
PackageVersion: '0.84'
```

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: inno
+FileExtensions:
+- sz
+AppsAndFeaturesEntries:
+- DisplayName: SzArchiver 0.84
+  Publisher: Super Flexible Software
+  DisplayVersion: 0.84
+  ProductCode: SzArchiver_is1
```
Comment (Trenly) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.installer.yaml:13]:
```suggestion
```
Not needed for this package as the normalization functions of winget will allow for a match using the PackageName, Publisher, and PackageVersion. Additionally, the DisplayVersion field should not be included if it matches the PackageVersion as it causes additional mapping on the backend which is undesirable.

File: manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
```
Comment (Trenly) [inline manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.yaml:5]:
```suggestion
PackageVersion: '0.84'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**